dev-requirements.in

```diff
@@ -7,6 +7,7 @@ wheel
 pip-tools
 pypandoc
 pytest-cov
+pytest-pep8
```

Comment:
> setup.cfg uses the `--pep8` args to pytest, which requires the pytest-pep8 package.

`--pep8` option is provided by [pytest-pycodestyle 1.0.6](https://pypi.org/project/pytest-pycodestyle/1.0.6/).

So, I think that development dependency updates may be revert.

Can you try `pytest --pep8` in your environment without pytest-pep8?

Reply:
Interesting, because until I installed pytest-pep8 I would get these errors.
```
$ pytest
usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --pep8
  inifile: /home/caseyp/git/pip-licenses/setup.cfg
  rootdir: /home/caseyp/git/pip-licenses

$ pip freeze | grep pep8
autopep8==1.4.3
pep8==1.7.1
$ pip freeze | grep pytest
pytest==4.2.1
pytest-cache==1.0
pytest-cov==2.6.1
pytest-pycodestyle==1.4.0
pytest-runner==4.2

Reply:
Ah, Maintainer has been replaced in [pytest-pycodestyle 1.4.0](https://pypi.org/project/pytest-pycodestyle/1.4.0/). And the `--pep8` options provided are also changed.

I am pinning 1.0.6 to support both Python 2.7 and 3+.

Can you create an environment with the [dev-requirements.txt file of the master branch](https://github.com/raimon49/pip-licenses/blob/master/dev-requirements.txt) and try `pytest --pep8` again?

e.g.)

```bash
$ pip install -r dev-requirements.txt
```

Or when restoring with pip-tools.

```bash
$ pip-sync dev-requirements.txt
```

Reply:
Ah, yes. If I downgrade that to the versions specified in `dev-requirements.txt`, `pytest --pep8` works fine. I've removed that commit and rebased the branch without it.